src/App.tsx

```diff
@@ -51,6 +54,11 @@ function App() {
           darkmode={darkmode}
           setDarkmode={(data: any) => {
             setDarkmode(data);
+            setSelectedMapStyle(getDefaultMapStyleForMode(data))
```

Comment:
Two things, `getDefaultMapStyleForMode` is too long of a name and needlessly specific/personalized, I try to stay as simple and generic as possible to stop people deriving non-existent meaning from function names, just a readability point.

Also, I think it would be good to check if the current style is set to satellite or not, as if the use has already switched to the satellite style, if they change themes, they likely don't want the map style changed.